serviceAccount/ServiceAccount.psm1

```diff
@@ -0,0 +1,1211 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Could you add a README.md to this folder, and include the topics mentioned [here](https://github.com/Microsoft/adfsOpenSource#-documentation). 

Also, could you update the README.md on the repo root to include a link to this project